web/src/modals_util.ts

```diff
@@ -19,6 +19,16 @@ export const custom_time_unit_values = {
     },
 };
 
+export function valid_custom_time_input(input: number): boolean {
```

Comment:
Added this based on this review feedback: https://github.com/zulip/zulip/pull/26109#discussion_r1312336717.